sw/device/tock/README.md

```diff
@@ -0,0 +1,46 @@
+# Tock OS
+
```

Comment:
Apologies, I tired to request this before but looking back it may have got lost. Could you please add a single sentence description either of the first or second sentence that explains what is contained in this subdirectory, in a way that makes sense for someone unfamiliar with Tock.

A reasonable assumption when navigating to sw/device/tock is that you might find a complete copy of the Tock OS. It would be good to quickly dispel that misconception.

Reply:
Didn't quite manage a single sentence, but added to the description.

Reply:
Done.